dallinger/heroku/clock.py

```diff
@@ -28,9 +25,10 @@
 @scheduler.scheduled_job('interval', minutes=0.5)
 def check_db_for_missing_notifications():
     """Check the database for missing notifications."""
+    config = get_config()
```

Comment:
@MatthewWilkes you didn't actually load the config here. Also shouldn't the AWS keys be fetched from the config object rather than from the environment?